src/lib/env.js

```diff
@@ -40,21 +38,12 @@ export class Env {
     return parser.parse.call(parser, emit, data);
   }
 
-  _createEntries(lang, ast) {
-    const entries = Object.create(null);
-    const create = lang.src === 'qps' ?
-      createPseudoEntry : createEntry;
-
-    for (let i = 0, node; node = ast[i]; i++) {
-      const id = node.$i;
-      if (id in entries) {
-        this.emit('duplicateerror', new L10nError(
-         'Duplicate string "' + id + '" found in ' + lang.code, id, lang));
```

Comment:
really? Dang. I was hoping we would report when a context has duplicate, inter-resource dupes. Because otherwise that may lead to very hard to discover bugs.

Also, I believe BTO reports that, right?